File: python/knock012.py
'''
python knock012.py ../Data/hightemp.txt ../Data/col1.txt ../Data/col2.txt
'''

import sys

if __name__ == '__main__':
    col1 = open(sys.argv[2], "w")
    col2 = open(sys.argv[3], "w")
    for line in open(sys.argv[1]):
        spl = line.strip().split('\t')
        print >>col1, spl[0]
        print >>col2, spl[1]

File: ryosuke/java/knock_2015java/src/knock_2015java/Knock012.java
/**
 * 
 */
package knock_2015java;

import java.io.File;

/**
 * @author ryosuke
 *
 */
public class Knock012 extends Knock {
	public File inFile;
	public File outFile1;
	public File outFile2;
	
	/**
	 * 
	 */
	public Knock012() {
		inFile = hightemp;
		outFile1 = col1;
		outFile2 = col2;
	}

	/* (非 Javadoc)
	 * @see knock_2015java.Knock#start()
	 */
	@Override
	public void start() {
		for(String line : readFile(hightemp)){
			String[] spl = line.split("\t");
			printFile(outFile1, spl[0]+"\n", true);
			printFile(outFile2, spl[1]+"\n", true);
		}
		System.out.println("fin");
	}

}
